Source/PLCrashReporterConfig.m

```diff
@@ -32,6 +32,17 @@
 #import "PLCrashReporterConfig.h"
 #endif
 
+/** @internal
```

Comment:
Let's rewrite the comment here adding details that it is the default value of a configurable option.